### PR TITLE
Add our new logo

### PR DESCRIPTION
**Pull request by saraedum**
since the author told me
(https://github.com/standardese/standardese/issues/125#issuecomment-537864314)
to "use it any way you want", let's explicitly dual-license this as MIT
and CC-BY (@bresilla holds the copyright, the "author" of this commit.)
Whether the former really applies for this "code" is not
completely clear but the idea is to just stick to the license that
everything else in this repo has.

### CODE REVIEW

**Comment by saraedum** [comment]
@bresilla does this look ok for you?

**Comment by bresilla** [comment]
> @bresilla does this look ok for you?

Yeah. Perfect. :)

**Comment by bresilla** [comment]
@saraedum I think would look cooler if the logo of the organisation would be the PNG version. Because as of now, in a black-themed github (through Stylus plugin) the logo looks with a white background an not transparent one :)

Here it is the PNG version:
https://i.imgur.com/W3Qa08T.png

**Comment by saraedum** [comment]
I converted the one I uploaded with `convert` but I guess I got the transparencies wrong. Let me try again.

**Comment by saraedum** [comment]
@bresilla, does it look ok now?

**Comment by bresilla** [comment]
Yes. Looks perfect now :)
